### ISSUE TITLE
Starter project: Optimize map importer

### ISSUE THREAD

**Issue by dabreegster** [dabreegster/abstreet#270]
Importing all of London, https://github.com/dabreegster/abstreet/blob/7a0dbb28b1e711bb9a9f7265ff95e3ee411089df/map_model/src/make/parking_lots.rs#L141 took 40 seconds. It's easy to use `timer.parallelize` to speed this up. Running the importer on large areas, checking the logs for the slowest steps, and applying simple fixes like `parallelize` and `FindClosest` is a great way to learn a bit about the map importing process.

**Comment by at-tran** [comment]
Is this issue solved? If not I would like to take a crack at it.

**Comment by dabreegster** [comment]
@at-tran: I just reimported the largest maps, and the time is now almost totally spent building the [contraction hierarchies](https://github.com/easbar/fast_paths/) for pathfinding. All of the other inefficiencies have been fixed, so I'll close out this issue.

If you're interested in data structures, #54 is another optimization task that's a ~gentle introduction to the simulation layer. Or if you're particularly interested in map importing, simulation, graphics, etc, I could point you towards other good starter bugs.
